hack/helm/pre_helm.go

```diff
@@ -49,6 +51,13 @@ const annotationsDest = `  template:
       labels:
         control-plane: controller-manager`
 
+const (
+	envHTTPSSrc  = `value: "433"`
```

Comment:
`443` -> `8443`

Reply:
this is what is on kustomize, we replace this with a helm template and `8443` as default.

Reply:
so let's set it to `8443 from the start right?`

Reply:
you are right, fixed a8185c9